indico/modules/events/tracks/client/js/index.js

```diff
@@ -24,34 +23,48 @@
           }
         });
 
-        $trackList.sortable({
+        $(this).find('.track-list').sortable({
+          items: '.track-row',
+          handle: '.ui-i-box-sortable-handle',
           axis: 'y',
-          containment: 'parent',
           cursor: 'move',
           distance: 2,
-          handle: '.ui-i-box-sortable-handle',
-          items: '> li.track-row',
           tolerance: 'pointer',
           forcePlaceholderSize: true,
+          placeholder: 'track-placeholder',
+          connectWith: '.track-list',
           update: function() {
-            var sortedList = $trackList
-              .find('li.track-row')
+            const sortedList = $(this).find('li.track-row')
               .map(function() {
-                return $(this).data('id');
+                if ($(this).hasClass('track-group-box')) {
```

Comment:
`const $this = $(this);` and then use that - no need to construct a new jQuery object all the time